lib/pillar/type_convert/to_elixir.ex

```diff
@@ -55,6 +55,9 @@ defmodule Pillar.TypeConvert.ToElixir do
     datetime
   end
 
+  # DateTime('Europe/Moscow')
+  def convert("DateTime" <> _time_zone, value), do: convert("DateTime", value)
```

Comment:
That's why I named commit "fix", not "support" :) It will require `tz` or `tzdata`, is it ok? https://hexdocs.pm/elixir/1.12/DateTime.html#module-time-zone-database

Reply:
What do you mind about runtime check availability of tz or tzdata? 
If it exists in a project - OK and use, otherwise error with Logger.warn("Add tzdata to project, man")

In Pillar, this dependency can be added only to :dev, :test env

Reply:
How can I check availability of tz or tzdata?
Isn't it better to provide full support for main types out of box?

Reply:
if Tzdata available
```
iex> Code.ensure_compiled(Tzdata)
{:module, Tzdata}

```
Otherwise:
```
Code.ensure_compiled(Tzdata)
{:error, :nofile}
```
 > Isn't it better to provide full support for main types out of box?
I'm not sure, because there are already 3-4 dependencies and its bad. 

We want down low dependencies in this library for reusage in our production projects. For example in next major version there will be removed Mint dependency

Reply:
I found a better way:
```
DateTime.new(~D[2016-05-24], ~T[13:26:08.003234], "Europe/Moscow")
```
returns
```
{:error, :utc_only_time_zone_database}
```
if no timezones db available.

It is better than check `Tzdata`, because `Tzdata` is not the only timezones db available (`tz` is another choice)

Reply:
Btw I'm sure, that better to return an error, instead of ignoring timezone and return value in UTC

PR will be merged and fixed a few